elements/pfe-accordion/src/pfe-accordion-panel.scss

```diff
@@ -48,3 +48,7 @@ $LOCAL: accordion;
     @include pfe-panel-variables($theme: #{$context});
   }
 }
+
+:host([pfe-disclosure="true"][expanded]) {
```

Comment:
This is likely not necessary.  It should propogate down from the parent `pfe-accordion` where that variable is being set.

Reply:
Because the border-left color (gray to blue) and weight (1px to 4px) change when the disclosure is open I broke these properties out separately (i.e., border-left-color, border-left-width). I can try to keep the `border` shorthand property but it was getting very confusing because the colors and weights change independently. 

Also the values for those are in the `pfe-accordion-props` mixin which are in the `pfe-accordion-header.scss` and `pfe-accordion-panel.scss` so I think that this is the correct place to call these.

Reply:
Moved the header and panel styles into the mixins. Let me know what you think.

Reply:
And I should have said it before: thanks so much for reviewing especially while you're out! Always appreciate your feedback, @castastrophe !

Reply:
What I'm suggesting though is that you can set those variables in the pfe-accordion and they will get used inside the panel. Does the accordion parent have a selector you can target that indicates it's a disclosure? You can update the border left width variable there and use it in the accordion panel. It keeps you from having to update the same values in different files.